### ISSUE TITLE
Dashboard controls vanish at browser zoom >150%

### ISSUE THREAD

**Issue by jessover9000** [grafana/grafana#20741]
**Steps to reproduce**:
Zoom into any dashboard and see how the toolbar changes:
150% zoom and smaller
<img width="1260" alt="Screenshot 2019-11-28 at 13 47 03" src="https://user-images.githubusercontent.com/53754871/69807533-d19afd80-11e5-11ea-81e1-caec68cd679d.png">

vs larger than 150%
<img width="1275" alt="Screenshot 2019-11-28 at 13 47 15" src="https://user-images.githubusercontent.com/53754871/69807534-d19afd80-11e5-11ea-9205-e2634cd48541.png">

**Actual Result**:
Time picker, zoom and refresh buttons stay visible, but the others (Add panel, star, share, save dashboard, dashboard settings and cycle view) vanish. They are not in the hamburger menu that appears at that zoom level, either.

**Expected Result**
I'm not sure whether all missing buttons are important enough to be shown in this super zoom view. Adding panels, saving and settings do seem important to me though. I think it could be an option to show all the buttons in a second (left-aligned?) row unter the dashboard name instead of hiding some of them.

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)

**Environment**:
- Grafana version: 6.5
- Data source type & version: gdev testdata
- User OS & Browser: OSX / Chrome
